OAuth/ResourceOwner/Auth0ResourceOwner.php

```diff
@@ -68,10 +67,19 @@ protected function configureOptions(OptionsResolverInterface $resolver)
             return str_replace('{base_url}', $options['base_url'], $value);
         };
 
-        $resolver->setNormalizers(array(
-            'authorization_url' => $normalizer,
-            'access_token_url'  => $normalizer,
-            'infos_url'         => $normalizer,
-        ));
+        // Symfony <2.6 BC
+        if (method_exists($resolver, 'setNormalizer')) {
```

Comment:
What about using Symfony version comparsion instead? It would be for sure more redeable for end-developers.